submissions/blueprint-rant/blueprint-rant.md

```diff
@@ -0,0 +1,51 @@
+
+![](Generic tiling Solar Array blueprint)
+
+Balancers, *the black magic creations*, are another example of something that most players want to use, with fair reason, but don't want to learn and understand the maths and mechanics behind them. Figuring out the exact combination of belts, splitters and undergrounds to make the ratios perfect make sounds like a fantastic sunday afternoon for some people, but many would find it dull and unexciting. In both these scenarios, allowing the importing of blueprints will *likely* improve the players enjoyment of the game as they wont get stuck making monotonous blueprints or diving into the maths of balancers.
+
+![](Factorio balancer overview)
+
+### Blueprint library and copy/paste
+
+Once you have built a thing once in a world, its pretty common to need to build it again later, be it miner setups, train stops, train tracks, solar setups, bus layouts et cetera. This is especially compounded in multiplayer games, where each player using their own rail standards can cause chaos and destruction. In these situations, being able to copy, paste and share blueprints using the blueprint library, allows a much smoother and less chaotic experience. I believe the key difference here is, you are recreating soemthing you have already build in the given save, not importing something from somewhere else. As mentioned by *Rseding91* earlier, the ability to use blueprints can also so quickly ruin a multiplayer save, so the tradeoff here is important to think about.
+
+### Creativity in Factorio - Kovarex enrichment
+
+As a final note before my conclusion, I'd like to highlight just how much space for creativity there is in this game and the missed opportunities airising from blindly importing blueprints, using the [Kovarex enrichment process](https://wiki.factorio.com/Kovarex_enrichment_process). This late game uranium processing method allows you to increase the quantity of useful Uranium-235 using a cyclical process. I believe that it is probably one of the more blueprinted processes in the game due to the initial complexity it appears to have.
+
+![](Kovarex recipe from wiki)
+
+Recently on the [r/factorio](https://www.reddit.com/r/factorio/) subreddit, there was a flood of different Kovarex enrichment setup photos, each one trying to one up the last in creativity, efficiency and all out weirdness. I'd like to share a few of these here, to highlight just how many options there are in this game.
+
+https://www.reddit.com/r/factorio/comments/ju509t/my_noobish_try_at_kovarex_enrichment/
+
+https://www.reddit.com/r/factorio/comments/it53gn/so_i_built_a_kovarex_enrichment_process_setup/
+
+https://www.reddit.com/r/factorio/comments/jkmkyc/my_overkill_beaconed_buffered_uranium_processing/
+
+https://www.reddit.com/r/factorio/comments/hgb8zn/the_1_million_monkeys_method_of_kovarex_enrichment/
+
+https://www.reddit.com/r/factorio/comments/jj4nsl/my_take_on_kovarex_circle_nuketrain_violently/
+
```

Comment:
I think you need a little conclusion here would help to pull this section together again, and restate what you're saying. Something in the lines of: `if you simply imported a blueprint you would not have had the adventure of figuring out your own little working setup. And even if you don't think your setup compares to other setups on Reddit, you're building it for yourself, not the random people on reddit.`

Or if you don't want add a paragraph, you could just clearly state this in the intro paragraph of this section, as that idea is a little vague.